docker-compose.yml

```diff
@@ -7,19 +7,19 @@ proxy:
    - "8888:80"
    - "5555:5555"
   volumes:
-   - proxy/nginx/conf:/opt/openresty/nginx/conf
-   - proxy/nginx/lualib:/opt/openresty/nginx/lualib
-   - proxy/nginx/sites-enabled:/opt/openresty/nginx/sites-enabled
-   - proxy/nginx/sites-available:/opt/openresty/nginx/sites-available
+   - ./proxy/nginx/conf:/opt/openresty/nginx/conf
+   - ./proxy/nginx/lualib:/opt/openresty/nginx/lualib
+   - ./proxy/nginx/sites-enabled:/opt/openresty/nginx/sites-enabled
+   - ./proxy/nginx/sites-available:/opt/openresty/nginx/sites-available
 api:
   extends:
     file: base.yml
     service: api
-  build: api
+  image: jazzdd/alpine-flask
   net: container:proxy
   volumes:
-   - api/ceryx:/ceryx/ceryx
-   - api/bin:/ceryx/bin
+   - ./api:/app
+  command: "-d"
```

Comment:
Ok, just checked. I don't believe it would be a good idea to have two processes that are not related in the same container. We should either keep the existing image here, or use the official Alpine flavor [`python2.7.11-alpine`](https://hub.docker.com/_/python/)